src/udx.c

```diff
@@ -26,7 +26,9 @@
 
 static uint32_t
 random_id () {
-  return 0x10000 * (rand() & 0xffff) + (rand() & 0xffff);
+  uint32_t id;
+  uv_random(NULL, NULL, &id, sizeof(id), 0, NULL);
+  return id;
```

Comment:
nice, the uv_random uses strong random entropy (which we don't really need, but doesn't hurt either!)

in the future we'd prob want to bulk alloc it, ie have a static buffer of 512b and fill that with random data on demand that we then slice off the ids from...